modules/openapi-generator/src/test/resources/2_0/issue1226.yaml

```diff
@@ -0,0 +1,71 @@
+swagger: '2.0'
+info:
+  description: 'blah'
+  version: 1.0.0
+  title: sample spec
+host: fake.site.com
+tags: []
+schemes:
+  - https
+paths:
+  /numberdata:
+    post:
+      summary: Get back a NumberHolder
+      description: ''
+      operationId: getNumberHolder
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters: []
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            $ref: '#/definitions/NumberHolder'
+  /numericqueryparams:
+    get:
+      tags:
+        - user
+      summary: a test route for numeric query params
+      description: ''
+      operationId: numericQueryParams
+      produces:
+        - application/json
+      parameters:
+        - in: query
+          name: int64
+          type: integer
+          format: int64
+          default: 9223372036854775807
```

Comment:
FYI. I got the following when testing the spring app generated from this spec:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project openapi-spring: Compilation failure
[ERROR] /private/tmp/spring/src/main/java/org/openapitools/model/NumberHolder.java:[20,24] integer number too large: 9223372036854775807
```

Appended "L" at the end of the default value fixes it.

For float, I fixed the issue by appending "F" to the default value.

Reply:
For CodegenProperty, I changed the code back to preserve the "l" suffix for this case. If you need me to change our existing code in AbstractJavaCodegen.java from adding "l" to "L" please let me know.

For CodegenProperty, I changed the code back to preserve the "f" suffix for this case. If you need me to change our existing code in AbstractJavaCodegen.java from adding "f" to "F" please let me know.